single_tech_samples/databricks/Cluster_Deployment/databricks/workspace.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adpWorkspaceName": {
+      "value": "samplewsname"
+    },
+    "adbWorkspaceLocation": {
```

Comment:
* adblocation instead of adbworkspaceloacation?

Reply:
I think we should leave it as workspace. If we can build other resources of databricks it can get confusing e.g. cluster/job